nr1.json

```
@@ -0,0 +1,6 @@
+{
+  "schemaType": "NERDPACK",
+  "id": "e7bd8d1a-7e58-4d99-9a31-eb3b4cc64c2d",
+  "displayName": "groundskeeper",
```

Depending where this shows up, I think it would be helpful to capitalize the displayName.
```suggestion
  "displayName": "Groundskeeper",
```